### DISCUSSION TITLE
FoxESS data node addresses for RS485 connection to W610 modbus interface

### DISCUSSION THREAD

**Discussion by EricMac1** [StealthChesnut/HA-FoxESS-Modbus#76] [Q&A]
Hi Does anybody have the data node list for extracting the information for HA. I'm after the USB address for the Battery Current. 1100??

**Comment by calum-mcfarlane** [comment]
https://github.com/StealthChesnut/HA-FoxESS-Modbus/wiki/Data-Register-Reference---H1-AC1
